### PR TITLE
add actuality template and change route name

### PR DESCRIPTION
**Pull request by AurelieFqt**

### CODE REVIEW

**Review by KevinRoig — DISMISSED** [review]
ok

File: src/Controller/GalleryController.php [hunk 1]
```diff
@@ -5,36 +5,63 @@
...
+    }
+
+    /**
+     * @Route("/actualites", name="actuality")
+     */
+    public function showArticles(ArticleRepository $articleRepository, ImageRepository $imageRepository): Response
+    {
+        $images = $imageRepository->findAll();
+        $articles = $articleRepository->findAll();
+        return $this->render('gallery/actuality.html.twig', ['articles' => $articles, 'images' => $images]);
```
Comment (RomainClair) [inline src/Controller/GalleryController.php]: You don't need to findAll images and send them to twig because you access your images through your article

**Review by RomainClair — CHANGES_REQUESTED** [review]

File: src/Controller/GalleryController.php (hunk 1 above)
Comment (AurelieFqt) [reply src/Controller/GalleryController.php]: effectively

**Review by KevinRoig — APPROVED** [review]
ok

**Review by RomainClair — APPROVED** [review]

**Pull request merged**